app/assets/javascripts/application/translation_workbench.js.coffee

```diff
@@ -130,6 +131,8 @@ class TranslationItem
 
   # @private
   build: ->
+    severity_texts = ['Minor', 'Neutral', 'Major', 'Critical']
```

Comment:
Better to move these strings to `en.yml`.

Reply:
These actually come from an enum on `Translation`. They probably should be moved to `translation_changes` though. I'll make that change, and should I pull the enum values from there or still use `en.yml`?

Reply:
I moved the enum to `TranslationChange` and made adjustments for that, and used the enum in the Translation Workbench.